pages/dashboard/index.js

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from "next/router";
+import { usePlan } from "@/utils/helpers";
+import SessionPreview from "@/components/SessionPreview";
+import styled from "styled-components";
+
+const DashboardContainer = styled.section`
+  min-height: 100vh;
+`;
```

Comment:
The same is true here: I think you can pass this style to the SessionPreview directly, or is this component used more than once with different heights?